CriticalMass/AppDelegate.swift

```diff
@@ -25,10 +25,21 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         window?.makeKeyAndVisible()
 
         appController.onAppLaunch()
+        handleLaunchArguments()
         return true
     }
 
     func applicationWillEnterForeground(_: UIApplication) {
         appController.onWillEnterForeground()
     }
+
+    private func handleLaunchArguments() {
+        #if DEBUG
+        let arguments = ProcessInfo().arguments
+
+        if arguments.contains("SKIP_ANIMATIONS") {
+            UIView.setAnimationsEnabled(false)
+        }
+        #endif
+    }
```

Comment:
There some more logic for XCTest in the `func application(_: UIApplication, didFinishLaunchingWithOptions _: [UIApplication.LaunchOptionsKey: Any]?) -> Bool ` 

Maybe we should move the test related setup code to one place even though it's not 100% related

Reply:
I tried my best in finding a good name to combine both. If you have another idea for the naming, feel free to make a suggestion 😄